src/Button/Button.stories.js

```diff
@@ -37,3 +37,77 @@ export const Block = Template.bind({})
 Block.args = {
   block: true,
 }
+
```

Comment:
Do you think it's worth getting rid of the legacy buttons in Storybook now or keep them there until we refactor them out?